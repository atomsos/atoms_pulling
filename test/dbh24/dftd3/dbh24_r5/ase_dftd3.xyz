6

F      -0.002850423565433      0.008761580731414     -2.761684177435182
C       0.118930820321591      0.122172844106443     -0.695077498777659
H       0.018130136305380      1.064140917276001     -0.658870833766096
H      -0.172596025639381     -0.472547092344373     -0.841587387273097
H       0.334852749198816     -0.430614155729204     -0.453951065904969
Cl      0.015583998760704      0.015512616663044      1.795776438362922
