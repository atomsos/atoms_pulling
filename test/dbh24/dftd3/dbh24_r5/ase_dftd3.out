  _________________________________
                                   
 |         DFTD3 V3.1 Rev 1        |
 | S.Grimme, University Bonn       |
 |         October  2015           |
 |   see dftd3 -h for options      |
  _________________________________

 Please cite DFT-D3 work done with this code as:
 S. Grimme, J. Antony, S. Ehrlich and H. Krieg,
 J. Chem. Phys. 132 (2010), 154104
 If used with BJ-damping cite also
 S. Grimme, S. Ehrlich and L. Goerigk,
 J. Comput. Chem. 32 (2011), 1456-1465
 For DFT-D2 the reference is
 S. Grimme, J. Comput. Chem., 27 (2006), 1787-1799
 For DFT-D3M or DFT-D3M(BJ) the reference is
 D.G.A. Smith, L.A. Burns, K. Patkowski, and 
 C.D. Sherrill, J. Phys. Chem. Lett. 7 (2016) 2197-2203

  files read :     
 ase_dftd3.xyz
C6 coefficients used:
           2  C6 for element            1
Z=  1 CN= 0.912     C6(AA)=    3.03
Z=  1 CN= 0.000     C6(AA)=    7.59
           5  C6 for element            6
Z=  6 CN= 0.000     C6(AA)=   49.11
Z=  6 CN= 0.987     C6(AA)=   43.25
Z=  6 CN= 1.998     C6(AA)=   29.36
Z=  6 CN= 2.999     C6(AA)=   25.78
Z=  6 CN= 3.984     C6(AA)=   18.21
           2  C6 for element            9
Z=  9 CN= 0.000     C6(AA)=    9.69
Z=  9 CN= 0.998     C6(AA)=    7.13
           2  C6 for element           17
Z= 17 CN= 0.000     C6(AA)=   92.35
Z= 17 CN= 0.997     C6(AA)=   90.40

#               XYZ [au]               R0(AA) [Ang.]  CN       C6(AA)     C8(AA)   C10(AA) [au] 
   1  -0.00539   0.01656  -5.21883   f     1.150    0.160         9.5       162.8      3413.1
   2   0.22475   0.23087  -1.31351   c     1.455    3.400        23.1       668.4     23679.8
   3   0.03426   2.01093  -1.24509   h     1.091    1.006         3.1        37.3       552.8
   4  -0.32616  -0.89298  -1.59037   h     1.091    2.003         3.0        36.6       541.8
   5   0.63278  -0.81374  -0.85784   h     1.091    2.016         3.0        36.6       541.8
   6   0.02945   0.02931   3.39353   cl    1.552    0.272        92.1      3841.5    196342.7

molecular C6(AA) [au] =       511.38
 --------------------------------------------------
 !! SOME DISTANCES VERY SHORT. CHECK COORDINATES !!
 --------------------------------------------------

           DFT-D V3
 DF pbe                                               
 parameters
 s6       :    1.0000
 s8       :    0.7220
 rs6      :    1.2170
 rs18     :    1.0000
 alpha6   :   14.0000
 alpha8   :   16.0000
 k1-k3    :   16.0000    1.3333   -4.0000
 Cutoff   :   95.0000 a.u.
 CN-Cutoff:   40.0000 a.u.

 Edisp /kcal,au:    -0.9228 -0.00147053

 E6    /kcal :    -0.1536
 E8    /kcal :    -0.7691
 % E8        : 83.35

 doing analytical gradient O(N^2) ...

 |G|=   1.6810052501964180E-003
gdisp time     0.0
 Cartesian gradient written to file dftd3_gradient
 normal termination of dftd3
